ConsoleApp/Vote.cs

```diff
@@ -0,0 +1,26 @@
+namespace ConsoleApp;
+
+
+
+public class Vote
+{
+    public string NameVote { get; set; }
+    public string CommaSepratedChoise1 { get; set; }
```

Comment:
there should be more than 2 options to choose between so this is not what was expected to do